content/docs/buildpack-author-guide/create-buildpack/caching.md

```diff
@@ -215,8 +222,15 @@ else
     # Determine if there has been a gem dependency change and install new gems to the bundler layer; re-using existing and un-changed gems
     echo "---> Installing gems"
     mkdir -p "$bundlerlayer"
-    echo -e "cache = true\nlaunch = true\nmetadata = \"$local_bundler_checksum\"" > "$bundlerlayer.toml"
-    bundle install --path "$bundlerlayer" --binstubs "$bundlerlayer/bin"
+    cat > "bundlerlayer.toml" <<EOL
+cache = true
+launch = true
+
+[metadata]
+checksum = "$local_bundler_checkssum"
+EOL
+    bundle config set --local path "$bundlerlayer" && bundle install && bundle binstubs --all --path "$bundlerlayer/bin"
```

Comment:
These are taken from https://github.com/buildpacks/docs/pull/291. Some of the code snippets were not updated.